bli-guides/_help_drivers/Halo.md

```diff
@@ -0,0 +1,78 @@
+ - *Scene*: Selecting the button causes selected scenes to fire.
+ - *Thermostat*: Thermostats work with three buttons that have different behaviors. The first one is MODE, in which the mode is chosen between those available in the thermostat, by moving the wheel. The second and third one are SETPOINTS, one for COOL and one for HEAT, where the wheel will change the setpoint of the selected mode.
+ - *Handle by custom macros*: This allows the user to create a custom behavior in *Macros*. The Halo offers events like "WHEEL TURNED", "PRESSED BUTTON", "ONLINE" and so on as well as commands to set title, subtitle and wheel level among others. An example of how to set this up can be found on the Khimo documentation website.
+
+The **Resources** dictate which resources will be affected by the button. The button can control any amount of resources, it is recommended to use resources of the same kind in a single button (eg: dimmable/non-dimmable), nevertheless, any kind is supported. 
+
+Each behavior acts differently towards mixed resource types when dealing with multiple resources.
+
+**Dimmers** can be of 3 types: dimmable with/without state and non-dimmable. When all of them are controlled together, moving the wheel means something different for each type. The dimmables with state will raise/lower with their current state as starting point, dimmables without state will raise/lower with the mean of the levels of all the other dimmables with state as starting point. Non-dimmables will turn on when the aforementioned mean is above 50% and off when it's below.
+
+Regarding **Shades**, they can have different capabilities, the ones taken into account here are, only raise/lower with/without state and set level with/without state. Mixing these together would mean that every shade behaves as a raise/lower without state. The same would happen with different types of tilt. Lastly, only one **Thermostat** and **Scene** is allowed per button.
+
+## Automatic configuration
+
+It is possible to create an automatic configuration for each Halo, relative to the room where it is located. It is enabled by checking the "Use automatic Halo configuration" option. BeoLiving Essential users always use the automatic configuration. 
+
+Setting this option will get the available scenes and resources of the current room and it will try to complete the 3 pages (8 buttons each) in this order:
+
+1. Local scenes that are not shown in dimmer,shade or thermostat view
+2. Local dimmer scenes, a master dimmer and buttons for each dimmer
+3. Local shade scenes, a master shade and buttons for each shade
+4. Local thermostat scenes, thermostat control with MODE and SETPOINTS
+5. Global scenes
+
+
+### Changelog
+
+#### v0.1 | 02/09/2021
+ - First version supporting the integration of the Beoremote Halo with the BLI.
```

Comment:
faltaria un "*Please, report any issue with this driver to: support+drivers@khimo.com*"